lib/query_adapters.ex

```diff
@@ -12,4 +12,21 @@ defmodule QueryAdapters do
 
     TTT.Repo.exists?(query)
   end
+
```

Comment:
First two functions in this file are public and shouldn't have underscores. I'd be tempted to make all of these functions public though. Seem like functions I'd want to call as a user of this module.

Reply:
All functions are now public notation